src/harness/util.py

```diff
@@ -202,6 +205,34 @@ def makePpaAndPalRecordsConsistent(ppa_record, pal_records, low_frequency,
     # Converting from defaultdict to dict
     pal_records[index] = json.loads(json.dumps(pal_rec))
 
+  return pal_records
+
+
+def makePpaAndPalRecordsConsistent(pal_records, ppa_record, low_frequency,
+                                   high_frequency, user_id,
+                                   fcc_channel_id="1"):
+  """Make PPA and PAL object consistent with the inputs
+
+    Args:
+      pal_records: (list) A list of PAL Records in the form of dictionary
+      which has to be associated with the PPA.
+      ppa_record: (dictionary) A dictionary containing PPA Record.
+      low_frequency: (number) The Primary Low Frequency for PAL.
```

Comment:
Can you add that it's in Hz? I've seen some Hz vs. MHz errors lately in other code :)